ledger/caching/src/test/lib/scala/com/daml/caching/MapBackedCacheForTesting.scala

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.caching
+
+import java.util.concurrent.ConcurrentMap
+
+// Never, ever use this in production code.
```

Comment:
It's a test class and it's aptly named, I don't think this extra warning is needed.